Cargo.toml

```diff
@@ -43,13 +43,14 @@ derivative = { version = "2.2.0", default-features = false, features = ["use_cor
 dirs-next = { version = "2.0.0", default-features = false }
 futures = { version = "0.3.17", default-features = false, features = ["alloc"] }
 http-types = { version = "2.12.0", default-features = false }
-manta-accounting = { git = "https://github.com/manta-network/manta-rs", tag = "v0.5.7", default-features = false, features = ["cocoon-fs"] }
-manta-crypto = { git = "https://github.com/manta-network/manta-rs", tag = "v0.5.7", default-features = false, features = ["getrandom"] }
-manta-parameters = { git = "https://github.com/manta-network/manta-rs", tag = "v0.5.7", default-features = false, features = ["download"] }
-manta-pay = { git = "https://github.com/manta-network/manta-rs", tag = "v0.5.7", default-features = false, features = ["bs58", "groth16", "network", "serde", "wallet"] }
-manta-util = { git = "https://github.com/manta-network/manta-rs", tag = "v0.5.7", default-features = false }
+manta-accounting = { git = "https://github.com/manta-network/manta-rs", default-features = false, features = ["cocoon-fs"] }
+manta-crypto = { git = "https://github.com/manta-network/manta-rs", default-features = false, features = ["getrandom"] }
+manta-parameters = { git = "https://github.com/manta-network/manta-rs", default-features = false, features = ["download"] }
+manta-pay = { git = "https://github.com/manta-network/manta-rs", default-features = false, features = ["bs58", "groth16", "serde", "wallet", "network", "parameters"] }
+manta-util = { git = "https://github.com/manta-network/manta-rs", default-features = false }
```

Comment:
we intend to get rid of the tag here, right?

Reply:
No, its just that the display change in manta-rs is in main but there is currently no release with that change, ill update this when there is 0.5.9 which should contain that part